govcd/vapp_network.go

```diff
@@ -223,3 +223,74 @@ func (vapp *VApp) RemoveAllNetworkFirewallRules(networkId string) error {
 	}
 	return nil
 }
+
+// UpdateNetworkStaticRouting updates vApp network static routes.
+// Returns pointer to types.VAppNetwork or error
+func (vapp *VApp) UpdateNetworkStaticRouting(networkId string, staticRoutes []*types.StaticRoute, enabled bool) (*types.VAppNetwork, error) {
+	task, err := vapp.UpdateNetworkStaticRoutingAsync(networkId, staticRoutes, enabled)
+	if err != nil {
+		return nil, err
+	}
+	err = task.WaitTaskCompletion()
+	if err != nil {
+		return nil, fmt.Errorf("%s", combinedTaskErrorMessage(task.Task, err))
+	}
+
+	return vapp.GetVappNetworkById(networkId, false)
+}
+
+// UpdateNetworkNatRulesAsync asynchronously updates vApp network static routes.
```

Comment:
Incorrect comment. Probably a copy/paste thing :) 
```suggestion
// UpdateNetworkStaticRoutingAsync asynchronously updates vApp network static routes.
```

Reply:
fixed